lib/shelley/test/data/cardano-node-shelley/shelley-genesis.yaml

```diff
@@ -40,7 +40,7 @@ genDelegs:
   8ae01cab15f6235958b1147e979987bbdb90788f7c4e185f1632427a:
     delegate: b7bf59bb963aa785afe220f5b0d3deb826fd0bcaeeee58cb81ab443d
     vrf: 4ebcf8b4c13c24d89144d72f544d1c425b4a3aa1ace30af4eb72752e75b40d3e
-updateQuorum: 5
+updateQuorum: 1
```

Comment:
could you explain why this change was needed?

Reply:
Ah! Took me some time to figure that out (still dreaming about a documentation of the genesis parameters one day :pray:). 
MIR certificates are used to transfer rewards to pre-registered stake key. This is not an operation that _anyone_ can do (that would be sort of problematic :sweat_smile:), but only the core delegates can (same goes for other "governance" operations like updating the protocol parameters). 

This `updateQuorum` parameters controls how many genesis delegate signatures are required for a governance type of request to be considered valid on-chain. Since we only have a single BFT leader in this setup, well, we can only have at most one signature! Thus I had to tweak this so that our BFT leader could act in full dictatorial mode :)